app/daemon.go

```diff
@@ -218,6 +238,97 @@ func environmentCheck() error {
 	return nil
 }
 
+func startWebhook(ctx context.Context, serviceAccount, kubeconfigPath, webhookType string) error {
+	logrus.Infof("Starting longhorn %s webhook server", webhookType)
+
+	var webhookPort int
+	switch webhookType {
+	case "admission":
```

Comment:
Can we use `types.WebhookTypeConversion` instead? the same for `conversion`.

Reply:
Fixed, thanks!